src/pages/LinkedAccounts/LinkedAccounts.tsx

```diff
@@ -0,0 +1,247 @@
+            value={account.key}
+            onChange={(e) => {
+              updateData(account, 'key', e.target.value);
+            }}
+          />
+        </div>
+        {account.linked ? renderUnlinkButton(account) : renderLinkButton(account)}
+      </div>
+    );
+  };
+
+  const renderLinkButton = (account: AccountMetaData) => {
+    return (
+      <Button
+        disabled={!isWorkspaceOwner || !account.key || running}
+        onClick={() => linkAccount(account)}
+      >
+        Link Account
+      </Button>
+    );
+  };
+
+  const renderUnlinkButton = (account: AccountMetaData) => {
+    return (
+      <Button
+        disabled={!isWorkspaceOwner || running}
+        onClick={() => {
+          setAccountToUnlink(account);
+          setModal(true);
+        }}
+      >
+        Unlink Account
+      </Button>
+    );
+  };
+
+  const linkAccount = async (account: AccountMetaData) => {
+    try {
+      setRunning(true);
+      const body = {
+        name: account.internalName,
+        key: account.key,
+      };
+      const res = await fetch(`/api/workspaces/${activeWorkspace.name}/linked-accounts`, {
+        method: 'POST',
+        body: JSON.stringify(body),
+      });
+      if (!res.ok) {
+        setError('Error linking account');
+        return;
+      }
+      updateData(account, 'linked', true);
+      await getAccounts();
+    } catch (error) {
+      setError(error);
+    } finally {
+      setRunning(false);
+    }
+  };
+
+  const unlinkAccount = async (account: AccountMetaData) => {
+    try {
+      setRunning(true);
+      const res = await fetch(
+        `/api/workspaces/${activeWorkspace.name}/linked-accounts/${account.internalName}`,
+        {
+          method: 'DELETE',
+        },
+      );
+      if (!res.ok) {
+        setError('Error linking account');
+        return;
+      }
+      updateData(account, 'linked', true);
+      await getAccounts();
+    } catch (error) {
+      setError(error);
+    } finally {
+      setRunning(false);
+    }
+  };
+
+  const renderModalContent = () => {
+    if (!accountToUnlink) return;
+    return <div>Are you sure you want to unlink your {accountToUnlink.externalName} account?</div>;
+  };
+
+  if (!data.length) {
+    return <div className="content-page">{error}</div>;
+  }
+  return (
+    <>
+      {modal && (
+        <Modal
+          content={renderModalContent()}
+          onCancel={() => {
+            setModal(false);
+            setAccountToUnlink(null);
+          }}
+          onSubmit={() => {
+            setModal(false);
+            unlinkAccount(accountToUnlink);
+            setAccountToUnlink(null);
+          }}
+        />
+      )}
+      <div className="content-page">
+        {renderHeader()}
+        {<div className="linked-accounts__error">{error}</div>}
+        <div className="linked-accounts">
+          {data.map((account) => (
+            <div key={account.internalName}>{renderAccount(account)}</div>
```

Comment:
Alongside each of the accounts, we will need to have a link that takes the user to documentation where they can see how to obtain this key. There is also something here you could link to if that's easier: https://dev.eodatahub.org.uk/docs/account-setup/